ee/tabby-ui/app/chat/page.tsx

```diff
@@ -136,6 +136,23 @@ export default function ChatPage() {
         document.execCommand('paste')
       } else if ((event.ctrlKey || event.metaKey) && event.code === 'KeyA') {
         document.execCommand('selectAll')
+      } else if (
+        event.key.toLowerCase() === 'l' &&
+        (event.ctrlKey === true || event.metaKey === true)
+      ) {
+        chatRef.current?.focus()
+        event.preventDefault()
+      } else {
+        const message = {
+          action: 'keydown',
+          key: event.key,
+          code: event.code,
+          ctrlKey: event.ctrlKey,
+          metaKey: event.metaKey,
+          shiftKey: event.shiftKey,
+          altKey: event.altKey
+        }
+        parent.postMessage(message, '*')
```

Comment:
detect the shortcut key and use `server.focusEditor`. see `onNavigateToContext` as an example.